doc/api/esm.md

```diff
@@ -1456,6 +1456,10 @@ _internal_, _conditions_)
 
 > Stability: 1 - Experimental
 
+> **Note: Do not rely on this flag. We plan to remove it once the
+> [Loaders API][] has advanced to the point that equivalent functionality can
+> be achieved via custom loaders.**
```

Comment:
```suggestion
> This flag will be removed when the [Loaders API][] offers equivalent
> functionality via custom loaders.
```

Reply:
That's fine. You can ignore my suggestion. Or maybe incorporate just a part of it and get rid of `Note:` and maybe the bold text?

```suggestion
> Do not rely on this flag. We plan to remove it once the
> [Loaders API][] has advanced to the point that equivalent functionality can
> be achieved via custom loaders.
```

Again, though, feel free to ignore it. My preference is less typographical variance and more concise wording, but every situation is also unique and has other considerations.

Reply:
The Loaders warning is in bold and starts with `Note:`. I want these to match, either both with `Note:`/bold or both without `Note:`/bold. Which is your preference?

Reply:
My preference is both without, but that is a non-blocking preference, so smashing the "Resolve conversation" button on this and not changing a thing is totally an option.

Reply:
I think it was @bmeck who put the loaders warning in bold, so giving him fair warning here that I’m removing `Note:` and bold from the Loaders warning unless he objects.